docs/source/upcoming_release_notes/1143-optics-notepad.rst

```diff
@@ -0,0 +1,32 @@
+1143 optics-notepad
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- added two new PVS to `OpticsPitchNotepad`
+- `MR2L3:PITCH:CCM:Coating1`
+- `MR2L3:PITCH:CCM:Coating2`
```

Comment:
```suggestion
- Added new PVS to ``OpticsPitchNotepad`` for storing the MR2L3 channel-cut monochromator (CCM) pitch position setpoints for its two coatings.
```

Is that accurate? I don't think we need to list the PVs here as it's an implementation detail